app/scripts/views/mixins/resume-token-mixin.js

```diff
@@ -40,10 +40,11 @@ define(function (require, exports, module) {
      * Get a stringified ResumeToken that can be passed along in an email
      *
      * @method getStringifiedResumeToken
+     * @param {Object} account
      * @returns {String}
      */
-    getStringifiedResumeToken () {
-      return this.getResumeToken().stringify();
+    getStringifiedResumeToken (account) {
+      return this.getResumeToken(account).stringify();
```

Comment:
I'm going to merge this because both you and @philbooth said "r+". I can open another PR with additional modifications if we should add some guards.